### DISCUSSION TITLE
Artifact when displaying more than 1 font awesome icon

### DISCUSSION THREAD

**Discussion by lcsqlpete** [twbs/bootstrap#37041] [Q&A]
If I display a single font awesome icon with the following code, the icon is displayed as expected.

```
<div class="row mb-4 fs-5 ke-grey-background py-2 d-print-none">
            <div class="col-md-12 text-center mt-2">
                <a href="#" class="me-4 text-primary" role="btn" id="close-btn">
                    <i class="fa fa-times"></i>
                </a>
            </div>
        </div>
```
However if I display a second icon in the same row like this:

```
<div class="row mb-4 fs-5 ke-grey-background py-2 d-print-none">
            <div class="col-md-12 text-center mt-2">
                <a href="#" class="me-4 text-primary" role="btn" id="close-btn">
                    <i class="fa fa-times"></i>
                </a>
                <a href="#" class="me-4 exportone text-primary" data-filename="download" data-tableid="mytable">
                    <i class="fas fa-file-export"></i>
                </a>
            </div>
        </div>
```
... the first icon has what appears to be an underline character between it and the second icon like this.

![Screen Shot 2022-08-27 at 11 43 30 AM](https://user-images.githubusercontent.com/8495380/187044018-a94061df-ca4c-4f9c-9049-b4fedf759bed.png)

The pattern continues if I add more icons: all icons except the last one have the underscore character after them.

Any ideas?

**Comment by mdo** [comment]
Whenever you need help from folks with a piece of code, it's always helpful to include a live demo via CodePen or JS Fiddle. As it stands now, I don't see any particular reason for the artifact unless you're getting some kind of underline from the link element. If you're using `href="#"`, change to a `<button>` element.

**Comment by KoalaBear84** [comment]
It looks like the 'artifact' is just the underline from the link. Try `text-decoration: none` to remove it.
